Framework/PythonInterface/mantid/py3compat/__init__.py

```diff
@@ -30,7 +30,7 @@
     from enum import Enum
 except ImportError:
     # use a compatability layer
-    from mantid.py3compat.enum import Enum  # noqa
+    from enum import Enum  # noqa
```

Comment:
This effectively does nothing and should be removed

I think the entire file can be removed actually?

Reply:
Well spotted, I think the file needs to stay to keep py3compat as a module as the mocking functionality differs between our python 3 versions?

Reply:
I agree with @DavidFair. I think the whole `py3compat` module can go as `unittest.mock` exists since Python 3.3 and that's all `py3compat.mock` uses.

Reply:
Okay, I tried removing the whole module and got some issues with `import unittest.mock` and just `import mock`, I'll give it another look

Reply:
I've tested Ubuntu / Windows and RHEL7 and we seem to have it on all

I suspect this was a lingering issue with the ghost of Python 2

Reply:
Seems that way, good to know :+1: